source/features/search-releases.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!hasAnyReleases()) {
+		console.debug('The repo does not have any releases.');
+
+		return false;
+	}
+
+	addSelectMenu();
+	registerEventListener();
+}
+
+const hasAnyReleases = () => select.exists('.release-entry') || select.exists('.release');
+
+const addSelectMenu = () => {
+	const container = select('.subnav');
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const selectedTag = getSelectedTagFromUrl();
+
+	container.append(
+		<div class="refined-github-search-releases float-right d-flex flex-shrink-0 flex-items-center mb-3">
```

Comment:
Our class name can start with `rgh-`

Reply:
Removed the class name from here as it was not being used.